backend/module/eCampCore/src/Service/InvitationService.php

```diff
@@ -116,4 +105,32 @@ public function rejectInvitation(string $inviteKey): Invitation {
 
         return new Invitation($camp->getId(), $camp->getTitle(), null, null);
     }
+
+    /**
+     * @throws EntityNotFoundException
+     * @throws NoAccessException
+     * @throws NotAuthenticatedException
+     * @throws EntityValidationException
+     */
+    public function resendInvitation(string $campCollaborationId): Invitation {
+        /** @var CampCollaboration $campCollaboration */
+        $campCollaboration = $this->campCollaborationRepository->find($campCollaborationId);
+        if (null == $campCollaboration) {
+            throw new EntityNotFoundException();
+        }
+        $user = $this->aclService->assertAuthenticated();
+        $this->aclService->assertAllowed($campCollaboration, Acl::REST_PRIVILEGE_PATCH);
+        if (CampCollaboration::STATUS_INVITED !== $campCollaboration->getStatus()) {
+            throw new EntityValidationException('Can only resend invitation if the status is '.CampCollaboration::STATUS_INVITED.', was: '.$campCollaboration->getStatus());
```

Comment:
To be discussed: Should all EntityValidationExceptions have a `messages` array instead of the `message` string? Like it is done e.g. in ContentTypeStrategyBase.php
This would help a lot with translating the backend error messages in the frontend.